bot.js

```diff
@@ -18,63 +18,84 @@
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
 
-// Use with Node.js
-// Requires discord.js, minestat.js (https://github.com/ldilley/minestat)
-
 const Discord = require('discord.js');
-const client = new Discord.Client();
 const auth = require('./auth.json');
+const serv = require('./minestat');
+
+const client = new Discord.Client();
 
 // Number of minutes to refresh server count after (recommended >= 1 min)
-const refreshEvery = 4;
-const refreshDelay = refreshEvery * 60000;
+const REFRESH_INTERVAL = 4 * 60 * 1000; // 4 minutes.
 
 // IP and port of Minecraft servers
 const servIP = {
     hypixel: 'mc.hypixel.net:25565',
-    // 'your title': 'your.ip:port',
 };
 
+let intervalId;
+
 // Executes as long as bot is online.
 client.on('ready', () => {
     console.log(`Logged in as ${client.user.tag}!`);
     console.log('Ready...');
     // Check status every 4 minutes
-    client.setInterval(checkStatus, refreshDelay);
+    intervalId = client.setInterval(checkStatus, REFRESH_INTERVAL);
 });
 
 
 // Executes upon reception of message.
 // States IPs of Minecraft servers respectively if '!ip' is typed into chat.
 client.on('message', async msg => {
     if (msg.content === '!ip') {
-        checkStatus();
-        for (const [name, ip] of Object.entries(servIP)) {
-            msg.reply(`${name}: ${ip}`);
+        
+        /**
+         * Possible race condition. 
+         * async/await evaluations are explicit through the 'AsyncFunction' and
+         * 'PausableFunction' constructs within the V8 runtime.
+         */
+
+        await checkStatus(); 
+
+        for (const name in servIP) {
```

Comment:
Could move this up before the call of `checkStatus()` so the response is instantaneous.